tests/integration/testdata/sync/infra/template-python-after.yaml

```diff
@@ -0,0 +1,48 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+
+Globals:
+  Function:
+    Timeout: 3
+
+Resources:
+  HelloWorldFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: after/Python/function/
+      Handler: app.lambda_handler
+      Runtime: python3.7
```

Comment:
nit: Are we able to set the runtime to 3.8 or 3.9? If so, 3.7 EOL will be less of a concern.

Reply:
Yes setting this to 3.8